traffic_ops/testing/api/v4/traffic_control_test.go

```diff
@@ -24,7 +24,6 @@ type TrafficControl struct {
 	ASNs                                              []tc.ASN                                `json:"asns"`
 	CDNs                                              []tc.CDN                                `json:"cdns"`
 	CacheGroups                                       []tc.CacheGroupNullable                 `json:"cachegroups"`
-	CacheGroupParameterRequests                       []tc.CacheGroupParameterRequest         `json:"cachegroupParameters"`
```

Comment:
Should these also be removed from the fixtures data?

Reply:
So it turns out, this didn't even have any fixtures data to remove.